packages/chain-mon/.eslintrc.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  extends: '../../.eslintrc.js',
+  extends: '../@eth-optimism/configs/.eslintrc.js',
```

Comment:
do we need the `..` prefix?

Reply:
since the eslint config files are in the given directory which is outside the chain-mon directory, I have added `..` prefix.

Reply:
Since this is a pnpm monorepo you can reference other packages by their package name
```suggestion
  extends: '@eth-optimism/configs/.eslintrc.js',
```